api/api.yaml

```diff
@@ -233,4 +274,20 @@ components:
           type: string
         website:
           description: Publicly reachable website address of the service provider
-          type: string
\ No newline at end of file
+          type: string
+    CredentialIssuers:
+      type: object
+      additionalProperties:
+        type: array
+        items:
+          $ref: "#/components/schemas/CredentialIssuer"
+    CredentialIssuer:
+      type: object
+      required:
+        - trusted
+        - serviceProvider
+      properties:
+        trusted:
+          type: boolean
+        serviceProvider:
+          $ref: "#/components/schemas/ServiceProvider"
```

Comment:
why not just return the DID?

Reply:
You need to determine in the UI if the issuer can be trusted. So, you must identity and contact them. Therfor we need to show them some contact information like a phone number and name.